mentat/config.py

```diff
@@ -53,16 +46,26 @@ class Config:
         converter=int_or_none,
         validator=validators.optional(validators.ge(0)),
     )
-    format: str = attr.field(
+    parser: Parser = attr.field(  # pyright: ignore
         default="block",
         metadata={
             "description": (
                 "The format for the LLM to write code in. You probably don't want to"
                 " mess with this setting."
             ),
-            "no_midsession_change": True,
         },
-        validator=validators.optional(validators.in_(parser_map)),
+        converter=parser_map.get,  # pyright: ignore
+        validator=validators.instance_of(Parser),  # pyright: ignore
+    )
+    no_parser_prompt: bool = attr.field(
+        default=False,
+        metadata={
+            "description": (
+                "Whether to include the parser prompt in the system message. Only set"
```

Comment:
A bit of a nit picky change but I think this should read 'This should only be set to true for fine tuned models'; the first time I read it I interpreted it as 'This is only set to true for fine tuned models'